### PR TITLE
WIP: Added CetoSwap tvl

### PR DESCRIPTION
**Pull request by tuco-gordian**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/ceto-swap exports TVL: 
        
 
 
    manta                     1.76 M
    
    total                    1.76 M

File: projects/ceto-swap/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const { getUniTVL } = require('../helper/unknownTokens');
+
+const factory = '0xf50c8e257ccf3e2b58651f78e3c2dc83446d9c47';
+
+module.exports = {
+  manta: {
+    tvl: getUniTVL({ factory, useDefaultCoreAssets: true, fetchBalances: true }),
```
[the remark this answers is not in the document]
Comment (g1nt0ki) [reply projects/ceto-swap/index.js:7]: this means, there is not enough liquidity (in your pools) for those tokens, and they are not listed on coingecko either, so no way for us to price those tokens.

[the remark this answers is not in the document]
Comment (g1nt0ki) [reply projects/ceto-swap/index.js:7]: yes

[the remark this answers is not in the document]
Comment (g1nt0ki) [reply projects/ceto-swap/index.js:7]: yes

[the remark this answers is not in the document]
Comment (tuco-gordian) [reply projects/ceto-swap/index.js:7]: I would like to clarify what "not enough liquidity" means. In which tokens should there be liquidity, and how much liquidity is required for TVL to be calculated?

[the remark this answers is not in the document]
Comment (g1nt0ki) [reply projects/ceto-swap/index.js:7]: liquidity against ETH/whitelisted stables

**Pull request merged**